Rakefile

```diff
@@ -1,10 +1,11 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
-require_relative 'config/application'
+require_relative "config/application"
 
 Rails.application.load_tasks
 
-task lint_ruby: ['lint:ruby']
-task lint_scss: ['lint:scss']
-task default: %i[spec lint_ruby lint_scss]
+if %w[development test].include?(Rails.env)
+
```

Comment:
Extra whitespace line. The linter should flag this I think.

Reply:
done https://github.com/DFE-Digital/register-trainee-teacher-data/pull/8/commits/c861f6aa8b7f3f8592200da2691fb542a5363e44